src/main/java/com/bookmark_manager/controller/BookmarkController.java

```diff
@@ -0,0 +1,46 @@
+package com.bookmark_manager.controller;
+
+import com.bookmark_manager.dto.BookmarkDto;
+import com.bookmark_manager.entity.Bookmark;
+import com.bookmark_manager.service.BookmarkService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/bookmarks")
+public class BookmarkController {
+  @Autowired
```

Comment:
Let's use constructor-based injection instead of property-based injection. It's recomended approach + it makes testing easier.
With lombok it's easy to do so just by adding `@RequiredArgsConstructor` and marking property as final